pkg/backup/backend/abs.go

```diff
@@ -12,27 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package backup
+package backend
 
 import (
 	"fmt"
 	"io"
 	"io/ioutil"
 
 	"github.com/coreos/etcd-operator/pkg/backup/abs"
+	"github.com/coreos/etcd-operator/pkg/backup/util"
 
 	"github.com/sirupsen/logrus"
 )
 
 // ensure absBackend satisfies backend interface.
-var _ backend = &absBackend{}
+var _ Backend = &AbsBackend{}
 
-type absBackend struct {
+// AbsBackend is the Azure Blob Storage backend.
+type AbsBackend struct {
```

Comment:
this should be private

Reply:
it must be exposed because backup.go in backup pkg uses it.